src/System.ServiceModel.Security/pkg/System.ServiceModel.Security.pkgproj

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <PreventImplementationReference>true</PreventImplementationReference>
```

Comment:
Meant to remove that, was trying various stuff to get the packages fixed before I talked to Eric St. John.  Will remove.